### ISSUE TITLE
Training Details of the LLama Model

### ISSUE THREAD

**Issue by HarryHe11** [ssu-humane/HerO#1]
Dear all, Thanks for the amazing work! I would like to inquire more about the training details of the veracity prediction model.
1. Trianing set: Is the dev set used in SFT? 
2. Code availability: Could you please open-source the construction scripts of the SFT dataset and the codes for SFT? 

Thanks for your response!

**Comment by lastdefiance20** [comment]
Hi @HarryHe11, thank you for your interest in our work!

1. For the **open-sourced** model that was evaluated on the dev set, we only used the **train set** for SFT of the model. However, for the competition(which used a hidden test set), we trained the model on the combined (train + dev) set. Further details are provided in our paper.

2. We currently don't have plans to open-source the training script, but it is quite simple to reproduce our training and can be reproduced using the Hugging Face TRL library for fine-tuning (We used it for training!). The training details are outlined in the paper.
> - Optimizer: Adam
> - Learning rate: 2e-5
> - Batch size: 128
> - Epochs: 2
> - LoRA configuration (If you use Lora for training): rank = 128, alpha = 256

The SFT dataset has already been released at Hugging Face, so you can train your model or modify our dataset to suit your needs!
https://huggingface.co/datasets/humane-lab/AVeriTeC-HerO

Feel free to reach out if you have any further questions!

**Comment by HarryHe11** [comment]
Thank you so much for your reply!
